src/components/MetricsPane/MetricsPane.jsx

```diff
@@ -316,6 +320,16 @@ const MetricsPane = ({ showMetricsPane, setShowMetricsPane, showLoadingIndicator
       return (
         <>
           <SelectedSiteHeader />
+          <SelectedSiteActionBar>
+            {getIsSiteSelected() && isMapView ? (
+              <ButtonSecondary onClick={() => zoomToSelectedSite({ displayedProjects, map })}>
+                <ZoomToSiteIcon /> &nbsp;Zoom
+              </ButtonSecondary>
+            ) : null}
+            <ButtonSecondary onClick={handleClearSelectedSampleEvent}>
+              <IconClose /> Clear
```

Comment:
I forgot to explain this! So ideally I would have left spacing to ButtonSecondary, but since we are using various sources for icons, some have their own built in spacing and some dont. This is one of the built-in-spacing ones.